tests/src/test/java/org/eclipse/hono/tests/registry/DeviceManagementIT.java

```diff
@@ -734,11 +735,19 @@ private static String assertLocationHeader(final MultiMap responseHeaders, final
         return generatedId;
     }
 
-    private static void assertRegistrationInformation(final Device response, final Device expectedData) {
+    private static void assertRegistrationInformation(final HttpResponse<Buffer> response, final Device expectedData) {
 
+        final JsonObject actualDeviceJson = response.bodyAsJsonObject();
+
+        // internal status is not decoded from JSON as users should not be allowed to change internal status
+        final JsonObject actualDeviceStatus = actualDeviceJson.getJsonObject(RegistryManagementConstants.FIELD_STATUS);
+        assertThat(actualDeviceStatus.getBoolean(RegistryManagementConstants.FIELD_AUTO_PROVISIONED)).isFalse();
```

Comment:
currently, the Device Registry Management API does not require the *status* property to be present in the response body. It also doesn't mention the *autoprovisioned* and * autoprovisioningnotificationsent* property at all.
So, IMHO we need to add these properties to the API spec, if they indeed can be returned by a registry implementation.

Reply:
Sure, makes sense.

I just became confused again: should the properties be camelCased or kebab-cased? I am asking since I noticed that _last-user_ is kebab-cased the rest is camelCased.

By majority decision counting the properties by case I think it is camelCased? 😄

Reply:
camel case, yes. `last-user` should be changed to `lastUser` FMPOV. But we will handle that in another issue: #2309